Anature/src/application/anatures/moves/moves/JustDamageDealing.java

```diff
@@ -8,6 +8,6 @@
 	@Override
 	public void activateMove(IAnature source, IAnature target)
 	{
-		target.takeDamage(calculateDamage(source, target, false));
+		target.takeDamage(calculateDamage(source, target, !isPhysicalAttack()));
```

Comment:
So all moves that are non-pysical are considered Special attacks? I made a couple of special attacks that are physical but also special.

Just making sure that you understand that the flag there is whether the move is special or not.

Reply:
yeah its either special or physical. Physical means use atk and def while special means use the special stats.

Reply:
Then why has `isPhysicalAttack` as the third argument for `calculateDamage` that flag is supposed to be identifying if the damage that will be calculated use attack and defense or special attacks and special defense.